tools/get_release_contributors.py

```diff
@@ -89,36 +96,36 @@ def get_login_or_realname(self, key, commit_hash):
         login = self._get_login(key, commit_hash)
         if login:
             return f'@{login}'
-        m = re.match('\A([^@<>]+) <.*>\Z', key)
-        if m:
-            return m.group(1)
-        logger.warning(f'unable to extract github login or real name from {repr(key)}')
+        matches = re.match(r'\A([^@<>]+) <.*>\Z', key)
```

Comment:
As far as I [read here](https://stackoverflow.com/questions/2241600/python-regex-r-prefix) r seems to say that "\" should be treated as literal `\`. Was the previous string incorrect?

Reply:
I think the previous was _almost_ incorrect, e.g. if that had been a lowercase `\a` then it'd be an ASCII bell, by mistake. In short see this Python expression:

```python
>>> "\A" == r"\A", "\a" == r"\a"
(True, False)
```

Reply:
Hmm. So it was correct by accident.
python3 shows me that both are equivalent (also using the python shell - thanks for pointing that out), so good you caught that.
```python
>>> r'\A([^@<>]+) <.*>\Z' == '\A([^@<>]+) <.*>\Z'
True
```